.gitattributes

```diff
@@ -0,0 +1 @@
+* text eol=lf
```

Comment:
Please revert.

Reply:
Is there a reason for this? Having to convert the EOLs on every file all the time is a hassle.

Reply:
It should be a different PR.